azure-devops/azext_devops/dev/team/wiki.py

```diff
@@ -0,0 +1,220 @@
+                                                 project=project,
+                                                 repository=repository)
+    wiki_params.repository_id = repository_id
+    if mapped_path:
+        wiki_params.mapped_path = mapped_path
+    if version:
+        from azext_devops.devops_sdk.v5_0.wiki.models import GitVersionDescriptor
+        version_descriptor = GitVersionDescriptor()
+        version_descriptor.version = version
+        wiki_params.version = version_descriptor
+    return wiki_client.create_wiki(wiki_create_params=wiki_params, project=project)
+
+
+def delete_wiki(wiki, organization=None, project=None, detect=None):  # pylint: disable=redefined-builtin
+    """Delete a wiki.
+    :param wiki: Name or Id of the wiki to delete.
+    :type wiki: str
+    """
+    organization, project = resolve_instance_and_project(detect=detect,
+                                                         organization=organization,
+                                                         project=project)
+    wiki_client = get_wiki_client(organization)
+    return wiki_client.delete_wiki(wiki_identifier=wiki, project=project)
+
+
+def list_wiki(organization=None, project=None, detect=None):
+    """List all the wikis in a project or organization.
+    """
+    organization = resolve_instance(detect=detect,
+                                    organization=organization)
+    wiki_client = get_wiki_client(organization)
+    return wiki_client.get_all_wikis(project=project)
+
+
+def show_wiki(wiki, open=False, organization=None, project=None, detect=None):  # pylint: disable=redefined-builtin
+    """Show details of a wiki.
+    :param wiki: Name or Id of the wiki.
+    :type wiki: str
+    :param open: Open the wiki in your web browser.
+    :type open: bool
+    """
+    organization, project = resolve_instance_and_project(detect=detect,
+                                                         organization=organization,
+                                                         project=project)
+    wiki_client = get_wiki_client(organization)
+    wiki_object = wiki_client.get_wiki(wiki_identifier=wiki, project=project)
+    if open:
+        webbrowser.open_new(url=wiki_object.remote_url)
+    return wiki_object
+
+
+def add_page(wiki, path, comment=_DEFAULT_PAGE_ADD_MESSAGE, content=None, file_path=None,
+             organization=None, project=None, detect=None):
+    """Add a new page.
+    :param wiki: Name or Id of the wiki.
+    :type wiki: str
+    :param path: Path of the wiki page.
+    :type path: str
+    :param content: Content of the wiki page. Ignored if --file-path is specified.
+    :type content: str
+    :param file_path: Path of the file input if content is specified in the file.
+    :type file_path: str
+    :param comment: Comment in the commit message of file add operation.
+    :type comment: str
+    """
+    if not content and not file_path:
```

Comment:
I can't create a page with empty content?

Reply:
Accidental empty page creation will be blocked. Since none of them are compulsory params. 

If user wants they can create by giving --content " "